kolibri/plugins/learn/assets/src/views/LibraryPage/NoResourcePage.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+
+  <div>
+    <h1>{{ $tr('header') }}</h1>
+    <p>
+      <KExternalLink
+        v-if="deviceContentUrl"
+        :text="$tr('adminLink')"
+        :href="deviceContentUrl"
+      />
+    </p>
+    <p v-if="showLearnerText">
+      {{ $tr('learnerText') }}
+    </p>
+  </div>
+
+</template>
+
+
+<script>
+
+  import urls from 'kolibri/urls';
+  import useUser from 'kolibri/composables/useUser';
+  import LearnAppBarPage from '../LearnAppBarPage';
+  import commonLearnStrings from '../commonLearnStrings';
+
+  export default {
+    name: 'NoResourcePage',
```

Comment:
This is more like "NoResourceComponent" now, I think? And then we could import this into the ContentUnavailablePage to act as the body and avoid the duplication here.